vilebot/src/main/java/com/oldterns/vilebot/handlers/user/FakeNews.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (C) 2019 Oldterns
+ *
+ * This file may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package com.oldterns.vilebot.handlers.user;
+
+import com.oldterns.vilebot.util.NewsParser;
+import org.apache.log4j.Logger;
+import org.pircbotx.hooks.types.GenericMessageEvent;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class FakeNews
+    extends NewsParser
+{
+    private static Logger logger = Logger.getLogger( FakeNews.class );
+
+    private static final String DEFAULT_CATEGORY = "canada";
+
+    private static final HashMap<String, URL> fakeNewsFeedsByCategory = new LinkedHashMap<>();
+    static
+    {
+        try
+        {
+            fakeNewsFeedsByCategory.put( "canada", new URL( "https://www.thebeaverton.com/rss" ) );
+            fakeNewsFeedsByCategory.put( "usa", new URL( "https://www.theonion.com/rss" ) );
+            fakeNewsFeedsByCategory.put( "belgium", new URL( "https://nordpresse.be/rss" ) );
+            fakeNewsFeedsByCategory.put( "france", new URL( "http://www.legorafi.fr/rss" ) );
+            fakeNewsFeedsByCategory.put( "india", new URL( "http://www.fakingnews.com/rss" ) );
+            fakeNewsFeedsByCategory.put( "russia", new URL( "https://fognews.ru/rss" ) );
+            fakeNewsFeedsByCategory.put( "serbia", new URL( "https://www.njuz.net/rss" ) );
+            fakeNewsFeedsByCategory.put( "venezuela", new URL( "http://feeds.feedburner.com/elchiguirebipolar" ) );
+            fakeNewsFeedsByCategory.put( "newzealand", new URL( "http://www.thecivilian.co.nz/rss" ) );
```

Comment:
To make it easier to update this list in the future, might be good to create a constant map of countries and then loop through it here

Reply:
What do you mean by a constant map? Is there an easier way to update a list than adding new entries in the HashMap?